<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grapher</class>
 <widget class="QFrame" name="grapher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grapher</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0" columnstretch="0,0">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="TabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="Graphing">
      <attribute name="title">
       <string>Graph Creator</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Graphh</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Photosphere">
      <attribute name="title">
       <string>Photosphere</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1" rowspan="2">
        <widget class="QOpenGLWidget" name="openGLWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>300</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Migration">
      <attribute name="title">
       <string>Migration Model</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QFrame" name="ModelCustomisation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="1">
           <widget class="QLineEdit" name="Area4Year">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="maxLength">
             <number>32767</number>
            </property>
            <property name="placeholderText">
             <string>N/A</string>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::CursorMoveStyle::VisualMoveStyle</enum>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="6">
           <widget class="QPushButton" name="MigrationModelButton">
            <property name="text">
             <string>Generate Model</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Area 4</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="Area3Year">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="maxLength">
             <number>32767</number>
            </property>
            <property name="placeholderText">
             <string>N/A</string>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::CursorMoveStyle::VisualMoveStyle</enum>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Area 1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="Area2Year">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="maxLength">
             <number>32767</number>
            </property>
            <property name="placeholderText">
             <string>N/A</string>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::CursorMoveStyle::VisualMoveStyle</enum>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="Area1Year">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="maxLength">
             <number>32767</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>N/A</string>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::CursorMoveStyle::LogicalMoveStyle</enum>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Area 3</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Area 2</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Area 5</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="Area5Year">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="maxLength">
             <number>32767</number>
            </property>
            <property name="placeholderText">
             <string>N/A</string>
            </property>
            <property name="cursorMoveStyle">
             <enum>Qt::CursorMoveStyle::VisualMoveStyle</enum>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGraphicsView" name="VideoContainer"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="eDNA">
      <attribute name="title">
       <string>eDNA Search</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="1">
        <widget class="QTableWidget" name="eDNAResults">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::PenStyle::SolidLine</enum>
         </property>
         <property name="rowCount">
          <number>10</number>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>Sample</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Score</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Classification</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item row="6" column="0" colspan="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QProgressBar" name="eDNAProgress">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
          <widget class="QPushButton" name="eDNAButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Classify Unknown Samples</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>500</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="UnknownSampleFolderPath">
           <property name="text">
            <string>:/</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="UnknownSampleFolderSelect">
           <property name="text">
            <string>Select Unknown Sample Folder</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="eDNADatabaseSelect">
           <property name="text">
            <string>Select eDNA Database</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="eDNADatabasePath">
           <property name="text">
            <string>:/</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Area1Year</tabstop>
  <tabstop>Area2Year</tabstop>
  <tabstop>Area3Year</tabstop>
  <tabstop>Area4Year</tabstop>
  <tabstop>Area5Year</tabstop>
  <tabstop>MigrationModelButton</tabstop>
  <tabstop>eDNADatabaseSelect</tabstop>
  <tabstop>UnknownSampleFolderSelect</tabstop>
  <tabstop>eDNAButton</tabstop>
  <tabstop>eDNAResults</tabstop>
  <tabstop>VideoContainer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
